.gitignore

```diff
@@ -65,3 +65,9 @@ coverage.xml
 !.gitignore
 .noseids
 Procfile
+
+# Local configuration file of udata
+udata.cfg
+
+# Local data directory
+data
```

Comment:
I use to put the configuration file and data directory one level up in my filesystem but if it can ease the first installation I'm OK with that.